File: Python/Program/Perusahaan.py
class Perusahaan:
    def __init__(self, NIB: str = "", namaPemilik: str = "", lokasi: str = ""):
        self.NIB = NIB
        self.namaPemilik = namaPemilik
        self.lokasi = lokasi

    # Getter
    def getNIB(self): return self.NIB
    def getNamaPemilik(self): return self.namaPemilik
    def getLokasi(self): return self.lokasi

    # Setter
    def setNIB(self, NIB): self.NIB = NIB
    def setNamaPemilik(self, nama): self.namaPemilik = nama
    def setLokasi(self, lokasi): self.lokasi = lokasi

File: Java/Program/Perusahaan.java
public class Perusahaan {
    private String NIB;
    private String namaPemilik;
    private String lokasi;

    public Perusahaan(String NIB, String namaPemilik, String lokasi) {
        this.NIB = NIB;
        this.namaPemilik = namaPemilik;
        this.lokasi = lokasi;
    }

    public Perusahaan() {
        this("", "", "");
    }

    public String getNIB() { return NIB; }
    public void setNIB(String NIB) { this.NIB = NIB; }

    public String getNamaPemilik() { return namaPemilik; }
    public void setNamaPemilik(String namaPemilik) { this.namaPemilik = namaPemilik; }

    public String getLokasi() { return lokasi; }
    public void setLokasi(String lokasi) { this.lokasi = lokasi; }
}
